scripts/get_plier_pathway_data.R

```diff
@@ -0,0 +1,23 @@
+# Script to download preprocessed pathway data from PLIER
+# repo and save it in a format that can be accessed in Python.
+base_url <- 'https://github.com/wgmao/PLIER/blob'
+commit <- '5351171779d32194dcacedb8e0e804021ea75fad'
+output_dir <- file.path(getwd(), 'data', 'pathway_data')
+
+canonical_file <- sprintf('%s/%s/data/canonicalPathways.rda?raw=true',
+                          base_url, commit)
+save_filename = file.path(output_dir, 'canonicalPathways.rda')
+download.file(canonical_file, save_filename)
+load(save_filename)
+output_filename = file.path(output_dir, 'canonical_pathways.tsv')
+write.table(canonicalPathways, file=output_filename,
```

Comment:
This is just an opinion thing, but I like to add row.names=FALSE to my write.table() functions. Otherwise you get an extra, unnamed column of (usually just ascending numbers) when you read in the file again.

Reply:
In this case the row names are the gene names, so I do want to keep them here. In general that's good to know, though!